### DISCUSSION TITLE
External legacy extensions support?

### DISCUSSION THREAD

**Discussion by Chirunos** [xiaoxiaoflood/firefox-scripts#209] [Legacy Development]
Is it supposed to work with other legacy (XUL/Bootstrap/Jetpack) extensions from other sources that aren't listed here?

**Comment by 117649** [comment]
If you dreaming about running XUL on current FF you are just dreaming.
FF no longer support XUL years ago.
Extensions must be developed against current versions of FF, If so it will run.

**Reply by Chirunos** [reply]
I am OK with using Nightly or Aurora/Developer Edition. Will these (legacy) extensions work with the tools provided here?
